JazzHands/IFTTTCAMediaTimingAnimation.m

```diff
@@ -0,0 +1,43 @@
+//
+//  IFTTTCAMediaTimingAnimation.m
+//  JazzHands
+//
+//  Created by Laura Skelton on 6/24/15.
+//  Copyright (c) 2015 IFTTT Inc. All rights reserved.
+//
+
+#import "IFTTTCAMediaTimingAnimation.h"
+
+@implementation IFTTTCAMediaTimingAnimation
+
+- (instancetype)initWithCAMediaTimingObject:(id<CAMediaTiming>)caMediaTimingObject
+{
+    if ((self = [super init])) {
+        _caMediaTimingObject = caMediaTimingObject;
+        _caMediaTimingObject.speed = 0;
```

Comment:
I do not really like the idea of setting `speed` to `0` because this will disable any automatic animation by default. Also, since the `CAMediaTiming` protocol models a hierarchical timing system, you probably cannot event work around this by adding another sublayer. Imagine the heart was supposed to beat all the time, regardless of the current scrolling.